documentation/src/main/asciidoc/topics/proc_configuring_hotrod_jmx.adoc

```diff
@@ -13,8 +10,19 @@ Rod client statistics.
 
 [source,java]
 ----
-include::code_examples/HotRodJmx.java[]
+ConfigurationBuilder clientBuilder = new ConfigurationBuilder();
```

Comment:
An ```import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;``` should make it clear we're not referring to the ConfigurationBuilder from core